pkg/planner/core/casetest/planstats/plan_stats_test.go

```diff
@@ -483,5 +483,6 @@ func TestPartialStatsInExplain(t *testing.T) {
 			output[i].Result = testdata.ConvertRowsToStrings(tk.MustQuery(sql).Rows())
 		})
 		tk.MustQuery(sql).Check(testkit.Rows(output[i].Result...))
+		require.NoError(t, dom.StatsHandle().LoadNeededHistograms(dom.InfoSchema()))
```

Comment:
Since only stats of predicates will be loaded, only stats of `a` from table `tp` is loaded into memory.
So for the first SQL, the stats of index `ic` and column `b` are not loaded into memory, becoming `unInitialized`. I use a `LoadNeededHistograms` to tell you that the actual stats is correct.